src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -52,7 +59,7 @@ const Container = styled.div`
   width: 100%;
   gap: 24px;
 
-  @media (min-width: 961px) {
+  @media (min-width: calc(960px + 1px)) {
```

Comment:
could you use the `MEDIA_WIDTHS.deprecated_upToMedium + 1` here?

Reply:
Shouldn't we use a non deprecated value? We do have breakpoints on theme that are widely used!

BTW, why do you had to do `+1` here? I've run into this issue recently while working on a different component and noticed we had some conditionals in React code regarding breakpoints. The issue was the programatic breakpoint used `<` instead of `<=` for min-width behaviour, that made it work unlike CSS, causing this `+1px` glitch.

I would really love if we could resolve that.